main.c

```diff
@@ -90,8 +93,16 @@ static void pointer_handle_motion(void *data, struct wl_pointer *wl_pointer,
 		seat_set_outputs_dirty(seat);
 	}
 
-	seat->x = wl_fixed_to_int(surface_x) + seat->current_output->logical_geometry.x;
-	seat->y = wl_fixed_to_int(surface_y) + seat->current_output->logical_geometry.y;
+	int x = wl_fixed_to_int(surface_x) + seat->current_output->logical_geometry.x;
+	int y = wl_fixed_to_int(surface_y) + seat->current_output->logical_geometry.y;
+
+	if (seat->state->edit_anchor) {
+		seat->anchor_x = seat->anchor_x + (x - seat->x);
```

Comment:
This can be simplified to

```c
seat->anchor_x += x - seat->x;
```

Reply:
Yep forgot it when I was trying to debug :sweat_smile: Thanks